erpnext/stock/doctype/packed_item/packed_item.py

```diff
@@ -63,38 +69,65 @@ def update_packing_list_item(doc, packing_item_code, qty, main_item_row, descrip
 	bin = get_bin_qty(packing_item_code, pi.warehouse)
 	pi.actual_qty = flt(bin.get("actual_qty"))
 	pi.projected_qty = flt(bin.get("projected_qty"))
+	packed_items_list.append(pi.idx)
 
-def make_packing_list(doc):
+def make_packing_list(doc, doctype=None, name=None, fieldname=None):
```

Comment:
Maybe use different names for the new variables, to distinguish them from `doc`.